addons/packages/cartographer/vendir.yml

```diff
@@ -2,12 +2,12 @@ apiVersion: vendir.k14s.io/v1alpha1
 kind: Config
 minimumRequiredVersion: 0.12.0
 directories:
-  - path: 0.2.2
+  - path: 0.3.0
     contents:
       - path: .
         githubRelease:
           slug: vmware-tanzu/package-for-cartographer
-          tag: v0.2.2
+          tag: v0.3.0
           disableAutoChecksumValidation: true
           assetNames:
           - "package.yaml"
```

Comment:
@cirocosta Instead of replacing 0.2.2 with 0.3.0, can you add a new path for `0.3.0`? That way we can be sure to keep the 0.2.2 version in sync as well. You can refer to the [kpack vendir file](https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/kpack/vendir.yml) for an example of this.

Reply:
done - rebased on [3ecc430](https://github.com/vmware-tanzu/community-edition/pull/3923/commits/3ecc430d7d7b83dd2ce5eb86a01aa513eaa0f0bf)